packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -119,33 +208,16 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
         right: 0,
         bottom: 0
       },
-      !isOpen &&
-        !isAnimating &&
-        isHiddenOnDismiss && {
-          visibility: 'hidden'
-        },
       isCustomPanel && classNames.custom,
       className
     ],
     overlay: [
       {
-        pointerEvents: 'none',
-        opacity: 1,
-        cursor: 'pointer',
-        transition: `opacity ${AnimationVariables.durationValue3} ${AnimationVariables.easeFunction1}`,
-        selectors: {
-          '@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none)': {
-            // For IE high contrast mode
-            backgroundColor: 'transparent'
-          }
-        }
-      },
-      isOpen && {
-        cursor: 'pointer',
-        pointerEvents: 'auto'
+        pointerEvents: 'auto',
+        cursor: 'pointer'
       },
-      isOpen && isAnimating && AnimationClassNames.fadeIn200,
-      !isOpen && isAnimating && AnimationClassNames.fadeOut200
+      isOpen && isAnimating && AnimationClassNames.fadeIn100,
+      !isOpen && isAnimating && AnimationClassNames.fadeOut100
```

Comment:
Decreased this to 100 vs 200 to fix a flash visual bug hard to notice caused by `isAnimating` changing a little earlier than the  `fadeIn` and `fadeOut` finish to animate.